README.md

```diff
@@ -14,4 +14,6 @@ The key scripts in this SDK are:
 
 To launch your project from the development environment you must first compile the project by running `make.cmd` (Windows), or opening a terminal in the SDK directory and running `make` (Linux / macOS).  You can then run `launch-game.cmd` (Windows) or `launch-game.sh` (Linux / macOS) to run your game.
 
-The `example` mod included in this repository provides the bare minimum structure to launch to the in-game main menu for the sole purpose of demonstrating the SDK.  See [Getting Started](https://github.com/OpenRA/OpenRAModTemplate/wiki/Getting-Started) on the Wiki for instructions on how to adapt this template for your own projects.  For common questions, please see the [FAQ](https://github.com/OpenRA/OpenRAModSDK/wiki/FAQ).
+The `example` mod included in this repository provides the bare minimum structure to launch to the in-game main menu for the sole purpose of demonstrating the SDK.  See [Getting Started](https://github.com/OpenRA/OpenRAModTemplate/wiki/Getting-Started) on the Wiki for instructions on how to adapt this template for your own projects.  For common questions, please see the [FAQ](https://github.com/OpenRA/OpenRAModSDK/wiki/FAQ).  See [Updating to a new SDK or Engine version](https://github.com/OpenRA/OpenRAModSDK/wiki/Updating-to-a-new-SDK-or-Engine-version) for a guide on updating your mod a newer OpenRA release.
+
+The OpenRA engine is made available under the [GPL](https://github.com/OpenRA/OpenRA/blob/bleed/COPYING) license, and any executable code developed by a mod and loaded by the engine (i.e. custom mod DLLs) must be released under a compatible license.  Your mod data files (artwork, sound files, etc) are not part of your mod's source code, so your are free to distribute these assets under different terms (e.g. allowing redistribution in unmodified form, but not for use in other works).
```

Comment:
You could include the OpenRAModSDK in `The OpenRA engine is made available under`.

Reply:
Also specify GPLv3 explicitly IMO.

Reply:
Where do yaml and campaign map/scripts go though?

Reply:
Map scripts are clear cut: [they must be gpl](https://www.gnu.org/licenses/gpl-faq.en.html#IfInterpreterIsGPL) (see paragraph 2).

Yaml is less clear: new files are not obligated to be gpl by the engine (they are input data, not linked or interpreted code), however by virtue of nobody ever saying otherwise the default mod yaml files have to be a treated as gpl ([which is allowed for data](https://www.gnu.org/licenses/gpl-faq.en.html#GPLOtherThanSoftware)).  But that is messy because the default mod yaml was translated from the proprietary ini data which was never specifically licensed for modification.

Reply:
I'd say to mention these explicitly in the readme as well.

Reply:
IMO the usage of similar names and values in the mods yaml files or anywhere else is a case of fair use. The yamls are rather a result of a trasformation for the specific and unique purpose of the project than a translation within a copy or recreation of the original work. Usage of basic ini data in gpl'ed yaml files complies to the [basic principles](https://en.wikipedia.org/wiki/Fair_use#U.S._fair_use_factors) of fair use IMO and the information used is publicly available too.